src/cpu/flexcpu/flexcpu_thread.cc

```diff
@@ -1001,13 +1006,21 @@ FlexCPUThread::populateDependencies(shared_ptr<InflightInst> inst_ptr)
     //       interface, to make these rules swappable?
 
     // Temporary virtual dependency rule to simulate in-order CPU mode.
```

Comment:
Expand this comment to explain what the dependency *is*?

Reply:
done